docs/nidcpower/examples.rst

```diff
@@ -1,6 +1,8 @@
 Examples
 ========
 
+Download examples for nidcpower and any required helper files at `nidcpower_examples.zip <https://github.com/ni/nimi-python/blob/master/examples/nidcpower_examples.zip>`_
```

Comment:
I think "and any required helper files" does more harm than good.
I also don't think it's necessary to make the URL readable in the rendered text.

Something like: "You can download all nidigital examples here" and the word here links to the file.

Reply:
Changed